src/Call.tsx

```diff
@@ -895,8 +895,8 @@ export class Call extends EventEmitter {
    * ```
    *
    * @param content - The message content
-   * @param contentType - The MIME type for the message. See {@link (CallMessage:namespace).ContentType}.
-   * @param messageType - The message type. See {@link (CallMessage:namespace).MessageType}.
+   * @param contentType - The MIME type for the message.
+   * @param messageType - The message type.
```

Comment:
Can we provide a link to the server API docs that lists the supported message types?

Reply:
There doesn't seem to be a Twilio Docs page/section that details this exactly. I made a ticket to do that in the next sprint.